<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UISetParam</class>
 <widget class="QWidget" name="UISetParam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UISetParam</string>
  </property>
  <widget class="QLabel" name="back_label">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>303</width>
     <height>295</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>74</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(113, 113, 113);</string>
   </property>
   <property name="text">
    <string>清晰度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>115</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(113, 113, 113);</string>
   </property>
   <property name="text">
    <string>麦克风</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>152</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(113, 113, 113);</string>
   </property>
   <property name="text">
    <string>摄像头</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="ratio_groupBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>63</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="HD_radio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>高清</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="SD_radio">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>标清</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="Audio_comboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>112</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="Video_comboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="close_toolButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>5</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/LoginWindow/images/btn_off.png);
border-image: url(:/LoginWindow/images/close.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>305</width>
     <height>297</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(171, 171, 171);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>305</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 204, 204);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(113, 113, 113);</string>
   </property>
   <property name="text">
    <string>弹   幕</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="bullet_checkBox">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>192</y>
     <width>70</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>225</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(113, 113, 113);</string>
   </property>
   <property name="text">
    <string>触发区域</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(113, 113, 113);</string>
   </property>
   <property name="text">
    <string>触发延时</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="tirgger_groupBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>215</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="btn_radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>70</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="all_radioButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>15</y>
      <width>70</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="delay_comboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>back_label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>ratio_groupBox</zorder>
  <zorder>Audio_comboBox</zorder>
  <zorder>Video_comboBox</zorder>
  <zorder>close_toolButton</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>bullet_checkBox</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>tirgger_groupBox</zorder>
  <zorder>delay_comboBox</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="loginwindow.qrc"/>
 </resources>
 <connections/>
</ui>
